ios/native/NativeProxy.mm

```diff
@@ -328,6 +329,22 @@ static id convertJSIValueToObjCObject(jsi::Runtime &runtime, const jsi::Value &v
   auto unregisterSensorFunction = [=](int sensorId) { [reanimatedSensorContainer unregisterSensor:sensorId]; };
   // end sensors
 
+  // keyboard events
+
+  static REAKeyboardEventObserver *kbObserver = [[REAKeyboardEventObserver alloc] init];
```

Comment:
I think that we should avoid abbreviations if it is not necessary, the `keyboardObserver` name is more informative.